backend/lambdas/api/spec.yaml

```diff
@@ -2079,6 +2079,7 @@ components:
               - php_sensio_security_checker
               - python_code_checker
               - technology_discovery
+              - cicd_tools
```

Comment:
do we also need to update the `artemis-scan.sh` script to add this plugin?

Reply:
Discussed this offline. For posterity, `artemis-scan.sh` now includes all categories by default and has an explicit block list of plugins not to run by default, so it does not need to be updated.